PYTHON_API_CONVENTIONS.md

```diff
@@ -0,0 +1,193 @@
+## Python API Coding Style Conventions
+
+Here is a list with some code conventions used by raylib in python:
+
+| Code element        |             Convention              | Example                                   |
+|---------------------|:-----------------------------------:|-------------------------------------------|
+| Variables           |             lower_case              | `delta_time = 0`, `player_age = 18`       |
+| Local variables     |             lower_case              | `player_position = Vector2(0, 0)`         |
+| Global variables    |             lower_case              | `window_ready = False`                    |
+| Constants variables |              ALL_CAPS               | `MAX_VALUE = 8`, `SCREEN_WIDTH = 800`     |
+| Definitions values  |              ALL_CAPS               | `MAX_BUILDINGS = 5`[^1]                   |
+| string values       |              always ""              | `output = "Hello"`, `"welcome"`           |
+| float values        |             always x.x              | `gravity = 10.0`                          |
+| Operators           |           value1 * value2           | `product = value * 6`                     |
+| Operators           |           value1 / value2           | `division = value / 4`                    |
+| Operators           |           value1 + value2           | `sum = value + 10`                        |
+| Operators           |           value1 - value2           | `res = value - 5`                         |
+| Class               |              TitleCase              | `class TextureFormat`                     |
+| Enum Class members  |              ALL_CAPS               | `PIXELFORMAT_UNCOMPRESSED_R8G8B8`         |
+| Class members       |              lowerCase              | `texture.width`, `color.r`                |
+| Functions           |    lowerCase & wordSeparationBy_    | `init_window()`, `update_camera_center()` |
+| Functions params    |              lowerCase              | `width`, `height`                         |
+| Ternary Operator    | result1 if condition else result2 | `print("yes" if value == 0 else "no")`  |
+[^1] like `macro definitions` of value in C
+
+Some other conventions to follow:
+ - **ALWAYS** initialize all defined variables.
+ - **use TABS / 4 spaces**.
+ - Avoid trailing spaces, please, avoid them
+ - Avoid using **semicolon** as you can
+ - Control flow statements always are followed **by a space**:
+```python
+if condition : value = 0
+
+while not window_should_close():
+    #Do something here!
+
+for i in range(NUM_VALUES): print(i)
+```
+ - All conditions checks are **always between parenthesis** but not boolean values:
```

Comment:
Okay that's big whaat.
- Why always?
- Why booleans are different?

50 and True will return 50. Should it be between parenthesis?

Reply:
Well, as you see the conversations haven't really been reviewed by electrostudio so yeah they aren't perfect, feel free to fork my doing and implementing your conversion 

p.s. and yeah the conventions aren't perfect because in the beginning they were made to mimic the raylib code style conventions but maybe follow PEP 8 would be better...